src/pudl_archiver/zenodo/api_client.py

```diff
@@ -0,0 +1,406 @@
+"""Core routines for archiving raw data packages on Zenodo."""
```

Comment:
Can you add args and return descriptions to the method docstrings in the module, please?

Reply:
Oh yeah I had kinda forgotten to update all these. Fixed now